fe/fe-core/src/main/java/com/starrocks/connector/RemoteFileOperations.java

```diff
@@ -290,4 +290,13 @@ public FileStatus[] listStatus(Path path) {
             throw new StarRocksConnectorException("Failed to list path %s. msg: %s", path.toString(), e.getMessage());
         }
     }
+
+    public FileStatus[] getFileStatus(Path[] paths) {
+        try {
+            return remoteFileIO.getFileStatus(paths);
```

Comment:
Does there exist  a cache for saving results of remoteFileIO.getFileStatus to reduce overwhelming duplicate requests?
if exists,  how we avoid reading stale result?
if not exists, can we add such a cache?

Reply:
no cache here, hard to distinguish stale.

Reply:
cache can be added in more high level.